src/app/shared/section-text/section-text.component.ts

```diff
@@ -0,0 +1,15 @@
+import { Component, Input, OnInit } from '@angular/core';
```

Comment:
Do not create too small components. I will slow down the app while rendering.
If adding some - use performance optimization.
Add OnPush strategy for this component.

Reply:
no fix